core/src/main/java/org/mskcc/oncotree/topbraid/OncoTreeVersionRepository.java

```diff
@@ -39,15 +39,15 @@
     private static final Logger logger = LoggerFactory.getLogger(OncoTreeVersionRepository.class);
 
     // NOTE we MUST order by release_date
-    private String query = "PREFIX oncotree-version:<http://data.mskcc.org/ontologies/oncotree_version/> " +
+    private String query = "PREFIX oncotree-version:<http://data.mskcc.org/ontologies/OncoTreeVersion#> " +
         "SELECT ?api_identifier ?graph_uri ?description ?release_date ?visible " +
         "WHERE { " +
-        "   GRAPH <urn:x-evn-master:oncotree_version> { " +
-        "       ?s oncotree-version:graph_uri ?graph_uri. " +
-        "       ?s oncotree-version:api_identifier ?api_identifier. " +
-        "       ?s oncotree-version:release_date ?release_date. " +
+        "   GRAPH <urn:x-evn-master:oncotreeversionlist> { " +
+        "       ?s oncotree-version:graphUri ?graph_uri. " +
+        "       ?s oncotree-version:apiIdentifier ?api_identifier. " +
+        "       ?s oncotree-version:releaseDate ?release_date. " +
         "       OPTIONAL{?s oncotree-version:description ?description.} " +
-        "       OPTIONAL{?s oncotree-version:visible ?visible.} " +
```

Comment:
just want to confirm the that the `oncotree-version:description` is supposed to remain `OPTIONAL` above here.

Reply:
Yes. When re-creating the oncotree-version ontology in a separate ontology using SHAQL constraints I thought about this and decided that maybe description could be optional, but everything else was required to be populated.

Every defined version *does* have a description. Maybe for consistency we should require all fields to be populated. Happy to change to that schema if it sounds better.

Reply:
I don't think it's necessary :) just wanted to double check.